api/src/main/java/edu/cornell/mannlib/vitro/webapp/dao/jena/DataPropertyDaoJena.java

```diff
@@ -360,8 +360,13 @@ public List getAllExternalIdDataProperties() {
     }
 
     protected boolean reasoningAvailable() {
-    	TBoxReasonerStatus status = ApplicationUtils.instance().getTBoxReasonerModule().getStatus();
-    	return status.isConsistent() && !status.isInErrorState();
+        try {
+            TBoxReasonerStatus status = ApplicationUtils.instance().getTBoxReasonerModule().getStatus();
+            return status.isConsistent() && !status.isInErrorState();
+        } catch (IllegalStateException e) {
+            log.debug("Status of reasoner could not be determined. It is likely disabled.", e);
```

Comment:
Suggest changing the logging level to `warn`.

Reply:
Same as above, I don't think this warrants a warn-level log because if the reasoner is unavailable it was likely intentional and the application will run as normal without it. This method is intended to determine if reasoning is available and returns false if not, that's not an error to me.